elastic/kibana
# [Alerting] Add alerting rule execution uuid to action event log documents

Resolves https://github.com/elastic/kibana/issues/121460

## Summary

Passes execution UUID to action executor using `action_task_params` saved object to be persisted inside the action execution event log documents.

## To Verify
1. Create a rule with some actions and let it run
2. View the event log documents and verify that the rule execution uuid stored inside the `provider: alerting` event log documents (in `kibana.alert.rule.execution.uuid`) is now also stored inside the `provider: actions` event log documents, also under `kibana.alert.rule.execution.uuid` and these uuids are the same for one execution cycle.


### Checklist
